common/scripted_effects/wc_sacrifice_scripted_effects.txt

```diff
@@ -0,0 +1,276 @@
+			if = {
+				limit = { PREV = { nzoth_offmap_decisions_enabled_trigger = yes } }
+
+				if = {
+					limit = { disliked_by_offmap = { type = offmap_nzoth } }
+					PREV = { multiply_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 1.25 } }
+				}
+				if = {
+					limit = { PREV = { liked_by_offmap = { type = offmap_nzoth } } }
+					PREV = { multiply_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 1.25 } }
+				}
+				if = {
+					limit = { PREV = { disliked_by_offmap = { type = offmap_nzoth } } }
+					PREV = { multiply_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 0.8 } }
+				}
+			}
+			# Yogg-Saron likes/dislikes
+			if = {
+				limit = { PREV = { yogg_saron_offmap_decisions_enabled_trigger = yes } }
+
+				if = {
+					limit = { disliked_by_offmap = { type = offmap_yoggsaron } }
+					PREV = { multiply_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 1.25 } }
+				}
+				if = {
+					limit = { PREV = { liked_by_offmap = { type = offmap_yoggsaron } } }
+					PREV = { multiply_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 1.25 } }
+				}
+				if = {
+					limit = { PREV = { disliked_by_offmap = { type = offmap_yoggsaron } } }
+					PREV = { multiply_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 0.8 } }
+				}
+			}
+		}
+	}
+}
+
+add_offmap_old_gods_currency_insanity_after_calculation_effect = {
+	PREV = {
+		if = {
+			limit = { old_gods_offmap_decisions_enabled_trigger = yes }
+
+			if = {
+				limit = { cthun_offmap_decisions_enabled_trigger = yes }
+
+				add_offmap_currency = {
+					offmap = offmap_cthun
+					value = local_received_offmap_old_gods_currency_insanity_variable
+				}
+			}
+			else_if = {
+				limit = { nzoth_offmap_decisions_enabled_trigger = yes }
+
+				add_offmap_currency = {
+					offmap = offmap_nzoth
+					value = local_received_offmap_old_gods_currency_insanity_variable
+				}
+			}
+			else_if = {
+				limit = { yogg_saron_offmap_decisions_enabled_trigger = yes }
+
+				add_offmap_currency = {
+					offmap = offmap_yoggsaron
+					value = local_received_offmap_old_gods_currency_insanity_variable
+				}
+			}
+		}
+	}
+}
+
+PREV_sacrifice_THIS_to_old_god_effect = {
+	caltulate_PREV_offmap_old_gods_currency_insanity_effect = yes
+	hidden_effect = { add_offmap_old_gods_currency_insanity_after_calculation_effect = yes }
+	if = {
+		limit = { PREV = { NOT = { society_member_of = twilights_hammer_cult } } }
+		PREV_public_sacrifice_THIS_effect = yes
+	}
+	else = {
+		death = { death_reason = death_missing }
+		character_event = { id = MNM.7506 }
+		PREV = {
+			show_scope_change = no
+			increase_visibility_minor_effect = yes
+			minor_soul_corruption_effect = yes
+			hidden_effect = { murder_bloodline_check_effect = yes }
+		}
+		from_society_sacrifice_root_effect = yes
+	}
+}
```

Comment:
`PREV_sacrifice_THIS_to_old_god_effect` says it uses `PREV` and `THIS` scopes, but `from_society_sacrifice_root_effect` uses `FROM` and `ROOT` scopes.

Reply:
It works, tho

But I can make new `PREV_society_sacrifice_THIS_effect`, if you want

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/74691890-c4464b80-51f5-11ea-9383-7b3602e11f78.png)
![image](https://user-images.githubusercontent.com/46576860/74691904-cf997700-51f5-11ea-9744-de401a90dbf4.png)

</details>

Reply:
Added!